### PR TITLE
update to unified build image

### PR DESCRIPTION
**Pull request by cseed**
make build, deploy per project

### CODE REVIEW

**Comment by cseed** [comment]
Also remove Dockerfile.pr-builder, environment.yml (now in hail repo) and clean up Makefile.

File: batch/environment.yml [hunk 1]
```diff
@@ -1,11 +0,0 @@
-name: hail-batch
```
Comment (danking) [inline batch/environment.yml]: we still want separate conda environments, right? Did this get moved elsewhere?

File: hail-ci-build-image [hunk 2]
```diff
@@ -1 +1 @@
-gcr.io/broad-ctsa/batch-pr-builder:953fd0b8928e8ab1e9fd1ef8396711d22c49ea9b37b09bccdee77f7b2493a1d4
+gcr.io/broad-ctsa/hail-pr-builder:b92d16e511405b281408164fe7dff452d93d91811ed9bcb263c5fb0de8df7a0c
```
Comment (danking) [inline hail-ci-build-image]: this isn't the hail-is/hail master SHA, is there a forthcoming PR into hail-is/hail that will provide the Dockerfile that generated this?

File: batch/hail-ci-build.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+source activate batch
+
+cd batch
```
Comment (danking) [inline batch/hail-ci-build.sh]: Changes at https://github.com/hail-is/hail/pull/4360 imply this should be changed similarly

**Comment by cseed** [comment]
I will re-open after things get sorted in the hail repo.

**Pull request closed without merging**